lte/gateway/python/magma/pipelined/app/inout.py

```diff
@@ -363,14 +363,26 @@ def _get_default_ingress_flow_msgs(self, dp):
             )
 
         # set a direction bit for outgoing (pn -> inet) traffic for remaining traffic
-        match = MagmaMatch()
+        # Passthrough is zero for packets from eNodeB GTP tunnels
+        ps_match_out = MagmaMatch(passthrough=0)
```

Comment:
Maybe define const NO_PASSTHROUGH_REG_VAL = 0 in register.py?  or just use REG_ZERO_VAL